tools/mt2mq/MT2MQ.xml

```diff
@@ -27,6 +28,7 @@
             </param>
             <when value="t">
                 <param name="ontology" type="hidden" value="NA" />
+                <param type="text" name="api_key" value="cab92f653d6d03c4aee4717918a6a925fc08" label="NCBI API key" />
```

Comment:
Upps, what is this? We can not include API keys here and in the tool. Please revoke this key if it is a real one - its now leaked :)

Reply:
You're right! Sorry, big mistake on my part. I just removed it from the code and we're working to replace the old leaked API key with a new one, which will not be shared :)

Reply:
@bgruening it should finally be ready to merge now! Sorry for the mistakes.